src/index.js

```diff
@@ -1,32 +1,26 @@
 import './style.css';
+import { addNewScore, getAllGameScores } from './modules/api.js';
 
-const scoreList = document.querySelector('.score-list');
+const refreshButton = document.querySelector('.refresh');
 const button = document.querySelector('.btn');
```

Comment:
- [ ] No actions from these buttons are related to the API since the file is basically empty. So all these buttons don't perform the expected actions. The submit button should hit the GET API endpoint and post user entries. Once more check the requirements and make sure your app is working as expected. When I click on those buttons right now nothing happen.